.gitignore

```diff
@@ -1,3 +1,4 @@
-
 simulation/TESS/
 simulation/__pycache__/
+.DS_Store
+*.csv
```

Comment:
I'm a little partial to multiple `.gitignore` files in subdirectories - it's gets too confusing. But either way, `.DS_Store` is already ignored in the root `.gitignore`, we shouldn't need it here.

Also, we allow (and have) csv files in other directories. Are we sure we don't need them here?

Reply:
I agree, I think this .gitignore should be combined with the root if it exists. 

I would lean towards not saving output *.csv from gridlabd simulations in Github. However, Let's just ignore config via the folder in root instead.